### DISCUSSION TITLE
Github education pack for high school students?

### DISCUSSION THREAD

**Discussion by steveiliop56** [community/community#138794] [GitHub Education]
### Select Topic Area

General

### Body

Hello,

I am currently a high school student and I want to get the education pack. When searching my school in the Github map thingy it find and registers my high school just fine. I am stuck in the proof of academic status page. I don't really know what proof it wants me to provide. Grades? Some paper from my school? Something else?

**Comment by ebndev** [comment]
Hi @steveiliop56, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `GitHub Education` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁

In the meantime, please feel encouraged to explore existing discussions in the category and utilize the search in case other users may have run into the same issue!

**Comment by davevad93** [comment]
Hi @steveiliop56 ,

As stated in the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-student), "***Documents that prove your current student status include a picture of your school ID with current enrollment date, class schedule, transcript, and affiliation or enrollment verification letter.***"

Therefore, you can use any of the above documents as proof to apply for the `Student Developer Pack`, the most important thing is that the document you provide shows a current date (preferably in `DD/MM/YYYY` or `MM/DD/YYYY` format) that can prove your student status.

**Reply by steveiliop56** [reply]
The drop down list also showed a school portal option, so I uploaded a screenshot of that. Not really sure if it will work though. I will ask my school if they provide some kind of paper.

**Reply by davevad93** [reply]
Hey @steveiliop56 , yes, ask your school if they can provide you with any document showing a current date, the name of the school and, of course, your first and last name; also make sure you have set up your GitHub profile correctly so you can apply without being rejected.

If you encounter any problems with your application you can consult the [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) where you can find a lot of useful information.
